app.rb

```diff
@@ -1,132 +1,89 @@
-require './book'
 require './classroom'
-require './teacher'
-require './student'
-require './person'
-require './rentals'
 
 class App
   attr_accessor :people, :rentals, :books
 
   def initialize
-    @people = []
-    @books = []
-    @rentals = []
+    @library = Library.new
+    @create_book = CreateBook.new(@library)
   end
```

Comment:
- [ ] As requested by the previous reviewer, Please fix all the issues occurring while running the app. When I start the app I get the below error. Please make sure to fix this and test the app functionalities in your local before going for another review.

![image](https://github.com/aima98/OOP-School-Library1/assets/83803414/5a7b9187-7494-4346-bcd3-2f82711c869c)